examples/nlp-case/run.py

```diff
@@ -67,7 +67,7 @@
 @click.option(
     "--no-cache",
     is_flag=True,
-    default=False,
+    default=True,
```

Comment:
shouldn't this get updated through the CI?

Reply:
It was: https://github.com/zenml-io/zenml/pull/2144/commits/d8ce54bb4c6e224e2081c61e0bd18832ca11e470